signicat/PublicKeys/1.xml

```diff
@@ -0,0 +1,104 @@
+         <Base_82>7487565870010927367880646662221312539929123988936652054057914198005996602363022383053438434070183447193692519510079849473955922129273484957761058343614903376564045479465581440717519845242749809415366552912605810654011033689216761262024529260265458167014377631696273237541134615448621625735486045702587755033371910336784478224884168606233434335421205143680764777750930270525758460816484434552671768027259815842834852353065693246598580583881984004562461683531683342879840136467142987205595004889007509749565588636834351900749861822186128348007288507207153008223813226341246329454169869745305174969541507367480338511312</Base_82>
+         <Base_83>10889674933319641087253248384577617641757165265207173906905638459626580935297161162463991046362623990217608499977400647243269436047751963486236110627711804028595258381444654599264363000011533657042806728489242021015622851834084754140968898079381332723894310688595161000733784170361618589292804349271114646766987097358478153436274516342021080500756239945365803550565433678022809302387080462215656493308001577934019238093264940622071036792168779181250042625039229488167017773107645734728959585295990789329294453351132837066678777490710955839129993815168859648556880046328465869111822175112802111848251918813753739192699</Base_83>
+         <Base_84>16829591681989522510735863275878354364810616820016172649664835152773887462267369206910992245292623237254109712721733816908946276254280620116537265089587383691565135725921408654741143800324266161140363989603515034836556546051952513485305715508490892133597770042225753671208334174990065708706136511739149363984416541946027328672594935447313734032247534112237927141569466772507065585690939425647583657730624409250790299353516423398166023082253314748611533152394074188897950870498374514532603907687892332158181799517078548904632303611285270678597653980914855735998313732810663218804474011425179986279955713451698084319828</Base_84>
+         <Base_85>4625754970226252870918582394520267793899692601908848476229272798113607387560528023023697458667131458800170032726258553664020775230618150665713057171691375163135034517706681226845729419102333057514209997380356352245693902711318828037273421198971584903536777560984185261130912844132623997352029116051654401958387209327211862299332873186664976459547292690640574601480110205936935955087170906725152201049522588585106158211678791228065683020898824422731595738359345289753426329018592353618004193083753845270915342657707567320413460807299645153183080842984313633549834363603243893188671237829458871665526552525535903081119</Base_85>
+      </Bases>
+   </Elements>
+   <Features>
+      <Epoch length="432000"></Epoch>
+   </Features>
+   <ECDSA>MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEg4voxTXZxqvb7qvWBaNtvO21gHDZXo/HMwPstxEzirAhFDhlTPQjLGpC/kMknVR1xHKSa2GdBpsU3sKn1uK1gg==</ECDSA>
+</IssuerPublicKey>
```

Comment:
You don't need multiple public keys (a 0.xml and a 1.xml). You are starting a new issuer, so one public/private key pair should be sufficient. Because this is demo, you may set the expiry date to 01-01-2030. That saves us some work next year.

Reply:
I changed the keys, and made sure they are valid till 2030.